.github/workflows/test.yml

```
@@ -133,6 +100,8 @@ jobs:
       - name: Check Rust Docs
         if: needs.changes.outputs.source == 'true'
         run: cd rust-doc && make docs
+      - if: needs.changes.outputs.source == 'true' || needs.changes.outputs.cue == 'true'
+        run: cargo vdev test-vrl
```

We should add a name for this step 🙂 
```suggestion
      - name:  VRL - Linux
        if: needs.changes.outputs.source == 'true' || needs.changes.outputs.cue == 'true'
        run: cargo vdev test-vrl
```

---

I also added some whitespace to increase readability to better catch that kind of thing.